### PR TITLE
id: 7229, comment: Update to include Social Sharing Icons

### PR DESCRIPTION
**Pull request by davidtrussler**
Ensures that Made with Dough uses the most recent version of Dough that includes the Social Sharing icons:
- Bumps version of Dough
- Adds SVG sprite

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -24,6 +24,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<!--[if ( !IE ) | ( gte IE 9 ) ]><!-->
+<%# render 'dough/shared/social_sharing_icon_sprite' %>
```
Comment (slaywell) [inline app/views/layouts/application.html.erb]: This is a bit peculiar, but I can see we do this in Frontend as well. Do you know what happens in IE9 without the conditional comments?

Comment (davidtrussler) [inline app/views/layouts/application.html.erb]: I *think* it is just so as to not load the SVG icons into a browser that doesn't understand them. (But yes, it is the same as frontend).

**Comment by slaywell** [comment]
:+1: but I might look at removing the conditional comments in the future

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (nmiddleweek) [inline app/views/layouts/application.html.erb]: IE8 and lower doesn't support SVG so we're not rendering the sprites.

**Comment by nmiddleweek** [comment]
:+1:

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (slaywell) [inline app/views/layouts/application.html.erb]: Yeah I know that, but just seems the wrong way around for progressive enhancement. Most of the enhancements for newer browsers will just get ignored in <IE9 so we don't have to specifically write conditional comments for everything. I was wondering if it actually causes any issues in IE8 and 9?

Comment (nmiddleweek) [inline app/views/layouts/application.html.erb]: I'm not sure, I don't think they always had conditionals around them, I remember something breaking when I first moved them into the base but that was at the bottom, I think Ben suggested wrapping them in the comments. I guess we'd need to remove them and re-test on all browsers.

I've tried find the commit but my comments do not give a reason which is annoying.

Comment (nmiddleweek) [inline app/views/layouts/application.html.erb]: If we are 100% certain that we will never ever use the SVG sprites in IE8 and below, then to me it makes sense to leave them un-rendered in that situation.

**Pull request merged**